sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_helpers.cs

```diff
@@ -30,7 +30,7 @@ public partial class WebPubSubServiceClient
         internal static byte[] s_group = Encoding.UTF8.GetBytes("webpubsub.group");
 
         /// <summary>
-        /// Creates a URI with authentication token.
+        /// Creates a URI with authentication token for clients who connect to "/clients/hubs/{hub}".
```

Comment:
`clients who connect to /clients/hubs/{hub}` it sounds a little technical detail, how about change it to `for WebSocket clients` and another one to `for WebSocket clients and MQTT clients`?